### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by originated**

### CODE REVIEW

File: src/app/locales/it.json [hunk 1]
```diff
@@ -0,0 +1,819 @@
...
+        "older": "Meno recente",
+        "or": "o",
+        "order_placed": "Ordine piazzato",
+        "password": "Password",
+        "payouts": "Pagamenti",
+        "permissions": "Permessi",
+        "post": "Post",
+        "post_as": "Pubblica come",
+        "posts": "Posts",
+        "powered_up_100": "Powered Up 100%%",
```
Comment (goldibex) [inline src/app/locales/it.json]: What's the guidance on translation of branded terminology? Are we leaving it as-is?

File: src/app/locales/it.json [hunk 2]
```diff
@@ -0,0 +1,819 @@
...
+        "since": "da",
+        "submit": "Sottoscrivi",
+        "power_up": "Power Up",
+        "submit_a_story": "Inserisci una Storia",
+        "tag": "Tag",
+        "to": "a",
+        "topics": "Argomenti",
+        "transaction_history": "Elenco delle transazioni",
+        "transfer": "Trasferisci",
+        "trending_topics": "Trending Topics",
```
Comment (goldibex) [inline src/app/locales/it.json]: argomenti emergenti?

File: src/app/locales/it.json [hunk 3]
```diff
@@ -0,0 +1,819 @@
...
+        "type": "Scrivi",
+        "unfollow": "Non seguire più",
+        "unmute": "Unmute",
+        "unknown": "Sconosciuto",
+        "upvote": "Upvote",
+        "upvote_post": "Upvote post",
+        "username": "Username",
```
Comment (goldibex) [inline src/app/locales/it.json]: Nome utente?

File: src/app/locales/it.json [hunk 4]
```diff
@@ -0,0 +1,819 @@
...
+        "faq": "FAQ",
+        "chat": "Steemit Chat",
+        "app_center": "Steemit App Center",
+        "api_docs": "Steemit API Docs",
+        "whitepaper": "Steem Whitepaper"
+    },
+    "main_menu": {
+        "hot": "hot",
+        "trending": "trending",
+        "payout_time": "payout time",
```
Comment (goldibex) [inline src/app/locales/it.json]: some untranslated terms here

File: src/app/locales/it.json [hunk 5]
```diff
@@ -0,0 +1,819 @@
...
+        "boost_payments": "Pagamenti da Promozione",
+        "authors": "Autori",
+        "curators": "Curatori",
+        "paste_a_youtube_or_vimeo_and_press_enter": "Copia un link da YouTube o Vimeo e premi Invio",
+        "there_was_an_error_uploading_your_image": "C'è stato un errore nel caricamento dell'immagine",
+        "raw_html": "HTML puro",
+        "please_remove_following_html_elements": "Rimuovere i seguenti elementi HTML dal tuo post: "
+    },
+    "witnesses_jsx": {
+        "witness_thread": "witness thread",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 6]
```diff
@@ -0,0 +1,819 @@
...
+        "replies_to": "Risposte a ",
+        "comments_by": "Commenti di"
+    },
+    "loginform_jsx": {
+        "you_need_a_private_password_or_key": "Hai bisogno di una password privata o di una chiave (non di una chiave pubblica)",
+        "cryptography_test_failed": "Test di crittofrafia fallito",
+        "unable_to_log_you_in": "Non è possibile effettuare il log in con questo browser.",
+        "the_latest_versions_of": "Le ultime versioni di ",
+        "are_well_tested_and_known_to_work_with": "are well tested and known to work with %(APP_URL)s.",
+        "due_to_server_maintenance": "Due to server maintenance we are running in read only mode. We are sorry for the inconvenience.",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 7]
```diff
@@ -0,0 +1,819 @@
...
+        "author_rewards_by": "Ricompense da autore da",
+        "replies_to": "Risposte a ",
+        "comments_by": "Commenti di"
+    },
+    "loginform_jsx": {
+        "you_need_a_private_password_or_key": "Hai bisogno di una password privata o di una chiave (non di una chiave pubblica)",
+        "cryptography_test_failed": "Test di crittofrafia fallito",
+        "unable_to_log_you_in": "Non è possibile effettuare il log in con questo browser.",
+        "the_latest_versions_of": "Le ultime versioni di ",
+        "are_well_tested_and_known_to_work_with": "are well tested and known to work with %(APP_URL)s.",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 8]
```diff
@@ -0,0 +1,819 @@
...
+        "account_name_should": "Il nome account",
+        "not_be_empty": "non deve essere vuoto.",
+        "be_longer": "essere più lungo.",
+        "be_shorter": "essere più corto.",
+        "each_account_segment_should": "Ogni segmento dell'account dovrebbe",
+        "start_with_a_letter": "iniziare con una lettera.",
+        "have_only_letters_digits_or_dashes": "avere solo lettere, cifre o trattini.",
+        "have_only_one_dash_in_a_row": "avere solo un trattino per riga. ",
+        "end_with_a_letter_or_digit": "terminare con una lettera o una cifra.",
+        "verified_exchange_no_memo": "You must include a memo for your exchange transfer."
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 9]
```diff
@@ -0,0 +1,819 @@
...
+    },
+    "settings_jsx": {
+        "invalid_url": "URL invalido",
+        "name_is_too_long": "Nome troppo lungo",
+        "name_must_not_begin_with": "Il nome non deve iniziare con @",
+        "about_is_too_long": "About is too long",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 10]
```diff
@@ -0,0 +1,819 @@
...
+        "location_is_too_long": "Luogo è troppo lungo",
+        "website_url_is_too_long": "L'url del sito è troppo lunga ",
+        "public_profile_settings": "Impostazioni profilo pubblico",
+        "private_post_display_settings": "Private Post Display Settings",
+        "not_safe_for_work_nsfw_content": "Not safe for work (NSFW) content",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

Comment (goldibex) [reply src/app/locales/it.json]: I actually already know this is "non sicuro per il lavoro" (NSPL)

File: src/app/locales/it.json [hunk 11]
```diff
@@ -0,0 +1,819 @@
...
+        "always_hide": "Nascondi sempre",
+        "always_warn": "Avverti sempre",
+        "always_show": "Mostra sempre",
+        "muted_users": "Utenti mutati",
+        "update": "Aggiornamento",
+        "profile_image_url": "Url immagine profilo",
+        "cover_image_url": "Url immagine di copertina",
+        "profile_name": "Mostra nome",
+        "profile_about": "About",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated (circa?)

File: src/app/locales/it.json [hunk 12]
```diff
@@ -0,0 +1,819 @@
...
+        "profile_location": "Località",
+        "profile_website": "Sito"
+    },
+    "transfer_jsx": {
+        "amount_is_in_form": "La quantita deve essere della forma 99999.999",
+        "insufficient_funds": "Fondi insufficienti",
+        "use_only_3_digits_of_precison": "Use only 3 digits of precison",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 13]
```diff
@@ -0,0 +1,819 @@
...
+        "send_to_account": "Invia a un account",
+        "asset": "Asset",
+        "this_memo_is_private": "Questo memo è privato",
+        "this_memo_is_public": "Questo memo è pubblico",
+        "power_up": "Power Up",
+        "submit": "Inserisci",
+        "basic": "Base",
+        "advanced": "Avanzato",
+        "convert_to_VESTING_TOKEN": "Converti in %(VESTING_TOKEN)s",
+        "balance_subject_to_3_day_withdraw_waiting_period": "Balance subject to 3 day withdraw waiting period,",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 14]
```diff
@@ -0,0 +1,819 @@
...
+        "withdraw_funds_after_the_required_3_day_waiting_period": "Preleva fondi dopo il periodo di attesa necessario di 3 giorni.",
+        "balance": "Bilancio",
+        "from": "Da",
+        "to": "A"
+    },
+    "userwallet_jsx": {
+        "conversion_complete_tip": "Si completerà su",
+        "in_conversion": "%(amount)s in conversione",
+        "transfer_to_savings": "Sposta nei Risparmi",
+        "power_up": "Power Up",
```
Comment (goldibex) [inline src/app/locales/it.json]: again, are these untranslated branded terms?

File: src/app/locales/it.json [hunk 15]
```diff
@@ -0,0 +1,819 @@
...
+        "savings": "RISPARMI",
+        "estimated_account_value": "Valore stimato dell'account",
+        "next_power_down_is_scheduled_to_happen": "Il prossimo power downo è programmato per avvenire",
+        "transfers_are_temporary_disabled": "I trasferimenti sono temporaneamente disabilitati.",
+        "history": "CRONOLOGIA"
+    },
+    "powerdown_jsx": {
+        "power_down": "Power Down",
+        "amount": "Amount",
+        "already_power_down": "You are already powering down %(AMOUNT)s %(LIQUID_TICKER)s (%(WITHDRAWN)s %(LIQUID_TICKER)s paid out so far). Note that if you change the power down amount the payout schedule will reset.",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 16]
```diff
@@ -0,0 +1,819 @@
...
+        "delegating": "You are delegating %(AMOUNT)s %(LIQUID_TICKER)s. That amount is locked up and not available to power down until the delgation is removed and a full reward period has passed.",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslated

File: src/app/locales/it.json [hunk 17]
```diff
@@ -0,0 +1,819 @@
...
+        "per_week": "That's ~%(AMOUNT)s %(LIQUID_TICKER)s per week.",
+        "warning": "Leaving less than %(AMOUNT)s %(VESTING_TOKEN)s in your account is not recommended and can leave your account in a unusable state.",
```
Comment (goldibex) [inline src/app/locales/it.json]: untranslted

File: src/app/locales/it.json [hunk 18]
```diff
@@ -0,0 +1,819 @@
...
+        "error": "Unable to power down (ERROR: %(MESSAGE)s)"
+    },
+    "checkloginowner_jsx": {
+        "your_password_permissions_were_reduced": "Your password permissions were reduced",
```
Comment (goldibex) [inline src/app/locales/it.json]: all these are untranslated

**Comment by originated** [comment]
@goldibex @sneak Please see updated, should have complete coverage.  As mentioned the remaining English terms are branded terms "POWER UP" & "STEEM POWER".  My understanding is they can't and shouldn't be translated.  Please advise going forward how to handle this.

**Pull request merged**